algorithmic_efficiency/workloads/librispeech_deepspeech/librispeech_pytorch/model.py

```diff
@@ -217,8 +223,13 @@ def reset_parameters(self) -> None:
       init.zeros_(self.weight)
       init.zeros_(self.bias)
 
-  def forward(self, input):  # pylint: disable=redefined-builtin
+  def forward(self, input):
+    # currently only GPU input is supported: added check below
+    # if not input.is_cuda:
+    #     raise ValueError("SyncBatchNorm expected input tensor to be on GPU")
```

Comment:
delete or uncomment?

Reply:
I have left it in so that it is easier for people to diff between this implementation and the original implementation?